Sources/Commands/Workspace.swift

```diff
@@ -623,7 +623,7 @@ public class Workspace {
             }
         }
 
-        return DependencyManifests(root: rootManifest, dependencies: dependencies.map{ $0.item })
+        return DependencyManifests(root: rootManifest, dependencies: dependencies.map{ ($0.item, dependencyMap[RepositorySpecifier(url: $0.item.url)]!) })
```

Comment:
I think the bang is correct... it is effective saying "I know this should exist, or crash", which is exactly right here.